csharp_webapi/11.QnAMaker/QnABot.bot

```diff
@@ -0,0 +1,38 @@
+﻿{
+    "name": "AspNetCore-QnA-Bot",
+    "description": "",
+    "secretKey": "",
+    "services": [
+      {
+        "id": "http://localhost:3978/api/messages",
+        "name": "AspNetCore-QnA-Bot",
+        "type": "endpoint",
+        "appId": "",
+        "appPassword": "",
+        "endpoint": "http://localhost:3978/api/messages"
+      },
+      {
+        "type": "qna",
```

Comment:
We cannot have the keys checked in. Remove these sections from the .bot file and add instructions in README to create the QnA Maker models and update the .bot file.